src/icons/Arrow.tsx

```diff
@@ -1,93 +1,77 @@
-import classnames, { animation, rotate } from 'classnames/tailwind'
+import classnames, {
+  animation,
+  dropShadow,
+  height,
+  transitionProperty,
+  width,
+} from 'classnames/tailwind'
 
-const arrowAnimation = (disabled?: boolean, vertical?: boolean) =>
+const arrowAnimation = (
+  pulseDisabled?: boolean,
+  openDisabled?: boolean,
+  open?: boolean
+) =>
   classnames(
-    animation(disabled ? undefined : 'animate-pulse-horizontal'),
-    rotate(vertical ? 'rotate-90' : undefined)
+    animation(
+      pulseDisabled
+        ? openDisabled
+          ? undefined
+          : open
+          ? 'animate-rotate-180'
+          : 'animate-rotate-0'
+        : 'animate-pulse-horizontal'
+    ),
+    transitionProperty('transition-all')
   )
+const svgInnerWrapper = classnames(
+  width('w-full'),
+  height('h-auto'),
+  dropShadow('drop-shadow-secondary')
+)
 
 interface ArrowProps {
-  disabled?: boolean
-  turnDown?: boolean
-  vertical?: boolean
+  openDisabled?: boolean
+  pulseDisabled?: boolean
```

Comment:
Maybe it's better to call `pulse`?

Reply:
Pulse is enabled by default, actually, let's disable it and enable with `pulse`